components/atom/input/src/inputs/TagInput/index.scss

```diff
@@ -0,0 +1,43 @@
+@import '~@schibstedspain/sui-atom-tag/lib/index';
+@import '~@schibstedspain/sui-theme/lib/index';
+
+.sui-AtomInput {
+  &-tag {
+    float: left;
+    margin-top: $m-s;
+  }
+
+  &-wrapper {
+    background: $c-white;
+    border-color: $c-gray-light;
+    border-style: solid;
+    border-width: 1px;
+    font-size: $fz-base;
+    overflow: hidden;
+    padding-left: $p-l;
+    padding-right: $p-l;
+    width: 100%;
+    display: flex;
+    flex-wrap: wrap;
+
+    input {
+      background: none;
+      border: 0;
+      float: left;
+      margin-left: $m-s;
+      flex-grow: 1;
+
+      &:focus {
+        box-shadow: none;
+        border: 0;
+        outline: none !important;
+      }
+    }
+
+    &.focus {
+      // added to SUI themes. To be change once SUI-theme is merged
+      box-shadow: 0 0 3px 0 #60acd0;
+      border: solid 1px #22749a;
+    }
```

Comment:
I believe that border and shadow width and color sould be a variable

Reply:
It should be a variable, but not a box-shadow one, but an outline one. Designers should decide how to call and define the "active" glow of an input or other elements. also called focus outline.

Reply:
@jcger